features/steps/givens/attributes.py

```diff
@@ -7,7 +7,10 @@
 
 @given("{attribute} = {value}")
 def step_impl(context, attribute, value):
```

Comment:
I assume my comment above could have more consequences. I'm open for discussion.

Reply:
I agree. That would be more readable.

This discussion spans across multiple pull requests. Since the remaining implementation of this pull request are good, I'd suggest to move the discussion there and merge this PR.
https://github.com/buildingSMART/ifc-gherkin-rules/issues/90